### PR TITLE
complete api login doc

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/api.rst [hunk 1]
```diff
@@ -117,4 +117,70 @@ two firewalls sharing the same context must be defined:
             path:   /logout
             target: /admin
 
+New authentication strategy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case when the strategies already available in Open Orchestra does not
```
Comment (juchi) [inline en/developer_guide/api.rst]: do not

File: en/developer_guide/api.rst [hunk 2]
```diff
@@ -117,4 +117,70 @@ two firewalls sharing the same context must be defined:
...
+        }
+
+To use this strategy, send a request to : ``/oauth/access_token?foo=bar``.
+The response should return a json object looking like:
+
+.. code-block:: json
+
+    { 'access_token': 'bar' }
+
+You can then use the ``bar`` access token to query the api:
```
Comment (juchi) [inline en/developer_guide/api.rst]: i think the correct verb is "call" the api

**Comment by nicolasThal** [comment]
@juchi doc updated

**Comment by juchi** [comment]
:+1:

**Pull request merged**